Farm&Gun/Assets/Scripts/SceneManagement/SceneLoader.cs

```diff
@@ -10,12 +10,12 @@ private class LoadingMonoBehaviour : MonoBehaviour
 
     public enum Scene
     {
-        MainMenu,
-        Settings,
-        SampleScene, // TODO: When main scene is done, replace
-        LoadingScene,
-        GameOver,
-        FullScene
+        MainMenu, //0
```

Comment:
Instead of adding a comment you could just write MainMenu = 0, Settings = 1 and so on